controllers/docluster_controller.go

```diff
@@ -166,14 +166,15 @@ func (r *DOClusterReconciler) reconcileDelete(ctx context.Context, clusterScope
 	if loadbalancer == nil {
 		clusterScope.V(2).Info("Unable to locate load balancer")
 		r.Recorder.Eventf(docluster, corev1.EventTypeWarning, "NoLoadBalancerFound", "Unable to find matching load balancer")
+		controllerutil.RemoveFinalizer(docluster, infrav1.ClusterFinalizer)
 		return reconcile.Result{}, nil
 	}
 
 	if err := networkingsvc.DeleteLoadBalancer(loadbalancer.ID); err != nil {
 		return reconcile.Result{}, errors.Wrapf(err, "error deleting load balancer for DOCluster %s/%s", docluster.Namespace, docluster.Name)
 	}
 
-	r.Recorder.Eventf(docluster, corev1.EventTypeNormal, "LoadBalancerDeleted", "Deleted a instance - %s", loadbalancer.Name)
+	r.Recorder.Eventf(docluster, corev1.EventTypeNormal, "LoadBalancerDeleted", "Deleted an instance - %s", loadbalancer.Name)
```

Comment:
Should this be LoadBalancer instead?
```suggestion
	r.Recorder.Eventf(docluster, corev1.EventTypeNormal, "LoadBalancerDeleted", "Deleted an LoadBalancer - %s", loadbalancer.Name)
```

Reply:
Ah yeah totally, a prior copy-pasta.